app/src/main/java/com/example/pocketwatching/Activities/ProfileActivity.java

```diff
@@ -67,11 +77,17 @@ public void done(List<Wallet> objects, ParseException e) {
                         String walletAddress = userWallets.get(i).getWalletAddress();
                         getEthWallet(walletAddress);
                     }
+                } else {
+                    // is this how i should throw errors?
+                    try {
+                        throw new Exception("There was an error: " + e);
+                    } catch (Exception ex) {
+                        ex.printStackTrace();
+                    }
```

Comment:
As mentioned in our 1:1 earlier, you can show a Toast instead, which not only lets your user know what's going on, but also makes it easier to debug while testing your app instead of having to dig through Logcat for your logs.